cmake/SetupPackages.cmake

```diff
@@ -61,6 +61,9 @@ endif ()
 
 if (ENABLE_HIP OR ENABLE_EXTERNAL_ROCPRIM)
   find_package(RocPRIM)
+  blt_import_library(NAME roctx
```

Comment:
I think this should be in a separate conditional block (not inside the "OR ENABLE_EXTERNAL_ROCPRIM") - it should be guarded with ENABLE_ROCTX.